### PR TITLE
MS-903 make email inputs field width larger

### PR DESCRIPTION
**Pull request by cmedlandho**

### CODE REVIEW

File: apps/verify/views/confirm-email.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
           type="text"
           name="confirm-email"
           id="confirm-email"
-          class="form-control{{#className}} {{className}}{{/className}}{{#error}} invalid-input{{/error}}"
+          class="form-control form-control-3-4 {{#className}} {{className}}{{/className}}{{#error}} invalid-input{{/error}}"
```
Comment (jjrobothamho) [inline apps/verify/views/confirm-email.html]: Cool Looks alright, @sulthan-ahmed  there is no other fancy way of doing this in HOF is there?

File: apps/nrm/fields/index.js [hunk 2]
```diff
@@ -658,6 +658,7 @@ module.exports = {
   },
   'pv-contact-details-email-input': {
     mixin: 'input-text',
+    className: ['form-control form-control-3-4'],
```
Comment (jjrobothamho) [inline apps/nrm/fields/index.js]: nice

File: apps/verify/views/confirm-email.html (hunk 1 above)
Comment (sulthan-ahmed) [reply apps/verify/views/confirm-email.html]: No this is correct.  This is a custom page so @cmedlandho was right to enter this here for this specific scenario

File: apps/nrm/fields/index.js (hunk 2 above)
Comment (sulthan-ahmed) [reply apps/nrm/fields/index.js]: good work

**Pull request merged**